src/simulations/exceptions.py

```diff
@@ -14,13 +14,17 @@
 
 
 class MetaboliteNotFound(Exception):
-    """Thrown when a search for a given metabolite identifier does not yield any result."""
+    """
+    Thrown when a search for a given metabolite identifier does not yield any result.
```

Comment:
I would expect this to cause an error in pydocstyle.

Reply:
Yes, the service is not yet on par with the cookiecutter and pydocstyle is still missing